iron/distribution.yaml

```diff
@@ -3402,6 +3402,14 @@ repositories:
       url: https://github.com/daviddorf2023/multidim_rrt_planner.git
       version: iron
     status: maintained
+  multidim_rrt_planner-release:
```

Comment:
I don't believe that you should have a `-release` suffix here, it should replace the section above?

Reply:
You should be able to hand-modify the yaml file to what you expect it to look like and update this PR.  I believe next time you run bloom (for your next release), you may need to edit the tracks, as well.

Reply:
Thank you for your help! Didn't realize bloom was opening a new branch on my rosdistro fork. I'm getting a better understanding of the tool now. Should be fixed now.